### PR TITLE
feat(builtin-package): use new blueprint

### PR DESCRIPTION
**Pull request by justprosh**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -58,6 +58,9 @@ jobs:
       - run: npm run build
         working-directory: aqua
 
+      - name: Install ipfs
+        uses: ibnesayeed/setup-ipfs@master
```
Comment (nahsi) [inline .github/workflows/tests.yml:62]:
```suggestion
        uses: ibnesayeed/setup-ipfs@0.6.0
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -105,6 +105,9 @@ jobs:
         run: npm publish --access public
         working-directory: aqua
 
+      - name: Install ipfs
+        uses: ibnesayeed/setup-ipfs@master
```
Comment (nahsi) [inline .github/workflows/release.yml:109]:
```suggestion
        uses: ibnesayeed/setup-ipfs@0.6.0
```

File: builtin-package/package.sh [hunk 3]
```diff
@@ -3,18 +3,44 @@ set -o pipefail -o nounset -o errexit
 
 # set current working directory to script directory to run script from everywhere
 cd "$(dirname "$0")"
-SCRIPT_DIR="$(pwd)"
+PACKAGE_DIR="$(pwd)/../package"
+
+(
+    rm -rf $PACKAGE_DIR
```
Comment (folex) [inline builtin-package/package.sh:9]: it worries me a lot to see rm -rf next to pwd. Maybe do rm $PACKAGE_DIR/*?

File: builtin-package/package.sh [hunk 4]
```diff
@@ -3,18 +3,44 @@ set -o pipefail -o nounset -o errexit
...
 (
     echo "*** create builtin distribution package ***"
     cd ..
-    tar --exclude="package.sh" -f trust-graph.tar.gz -zcv --transform 's|builtin-package|trust-graph|' builtin-package
+
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        tar -czvf trust-graph.tar.gz -s '|package|trust-graph|' package
+    else
+        tar -czvf trust-graph.tar.gz --transform 's|package|trust-graph|' package
+    fi
```
Comment (folex) [inline builtin-package/package.sh:43]: Why do you need this? You can either rename `$PACKAGE_DIR` to be `trust-graph`, or add files to tar.gz without any directory at all, directly to tar's `/`

**Review by nahsi — APPROVED** [review]

**Pull request merged**